perl-lib/OESS/lib/OESS/DB/Interface.pm

```diff
@@ -86,6 +86,14 @@ sub fetch{
 
     push(@{$in_use},@{OESS::DB::Interface::circuit_vlans_in_use(db => $db, interface_id => $interface_id)});
 
+    
+    my $provisionable_bandwidth;
+    if($interface->{'cloud_interconnect_type'} eq 'azure-express-route'){
+       $provisionable_bandwidth = $interface->{bandwidth}*4;
+    }elsif(defined $interface->{'cloud_interconnect_type'}){
+      $provisionable_bandwidth = $interface->{bandwidth};
+    }
```

Comment:
Couple minor nit picky things:
1. `$interface->{cloud_interconnect_type}` is preferable to `$interface->{'cloud_interconnect_type'}`
    See [here](https://gist.github.com/jonstout/b6927c6de4531e01aea26d0f99a015b8#hashes) for general style points.
2. `if ($foo eq $bar) {` is preferable to `if($foo eq $bar){`
3. Your code isn't aligned properly. 4 spaces per-indentation level is good. Avoid tabs please.

Otherwise this is good.